public/tools.json

```diff
@@ -1518,5 +1518,14 @@
     "logo": "feather-logo.png",
     "alt": "",
     "tag": ["fonts", "typography", "icon", "icons", "iconography"]
+  },
+  {
+    "id": 162,
+    "title": "Open Peeps",
+    "url": "https://openpeeps.com/",
+    "description": "Hand-drawn illustrations of people of the world. Can you recreate yourself?",
+    "logo": "openpeeps.png",
```

Comment:
I do not see this image added to the repo? Did you perhaps forget to check it into source control?

Reply:
@Rico-Smit a friendly nudge.

Reply:
@schalkneethling  The img is attached and the conflict is resolved.
![open-peeps](https://user-images.githubusercontent.com/89764927/221663914-8fe57c6e-426d-458e-9a50-0966e8427398.png)